include/fmt/chrono.h

```diff
@@ -1214,7 +1214,7 @@ template <typename OutputIt, typename Char> class tm_writer {
     char buf[10];
     size_t offset = 0;
     if (year >= 0 && year < 10000) {
-      copy2(buf, digits2(to_unsigned(year / 100)));
+      copy2(buf, digits2(static_cast<size_t>(to_unsigned(year / 100))));
```

Comment:
`to_unsigned` is kinda redundant now that we have a cast, let's remove it.

Reply:
But `to_unsigned()` contains a ">= 0" check.

Would it not be possible, that:
-  `tm_year()` returns a negative number (line 1213)
- negative number divided by 100 results in a negative number (line 1217)
- casting a negative number to size_t results in a big positive number
- the big positive number would overrun (line 1217) the char array in the function `digits2` (format.h line 919/925)?

Reply:
No, this function (line 1217) is only called for ``year`` in range [0, 10000). Check on line 1216.

Reply:
ok